qa/tasks/cephfs/test_volume_client.py

```diff
@@ -705,9 +697,9 @@ def test_get_authorized_ids(self):
         # Check the list of authorized IDs and their access levels.
         if self.py_version == 'python3':
             expected_result = [('guest1', 'rw'), ('guest2', 'r')]
+        # TODO: remove as soon as Python 2 support is deprecated.
```

Comment:
this won't happen; ceph_volume_client.py is deprecated.

Reply:
True. I'll get rid of it.